### PR TITLE
Creat pull Request form LamTruong

### PR DESCRIPTION
**Pull request by LamTruongJS**

### CODE REVIEW

File: src2/App.css [hunk 1]
```diff
@@ -1,44 +0,0 @@
-.App {
-  text-align: center;
-  width: 100%;
-  height: 100%;
-  box-sizing: border-box;
-  padding: 0;
-  font-family: "ヒラギノ角ゴシック Pro", "Hiragino Kaku Gothic Pro", "メイリオ",
```
Comment (khalt00) [inline src2/App.css:7]: can you fix like this?
`  text-align: center;
  width: 100%;
  height: 100%;
  box-sizing: border-box;
  padding: 0;`

Comment (LamTruongJS) [reply src2/App.css:7]: i can fix

File: src2/Components/Header/HeaderLower/HeaderLower.tsx [hunk 2]
```diff
@@ -1,80 +0,0 @@
...
-          onClick={handleToggleModalAddProject}
-          sx={{
-            color: "#fff",
-            width: "28px",
-            height: "100%",
-            padding: "5px",
-            cursor: "pointer",
-          }}
-        />
-      </Grid>
```
Comment (LamTruongJS) [inline src2/Components/Header/HeaderLower/HeaderLower.tsx:71]: test comment

File: src2/App.tsx [hunk 3]
```diff
@@ -1,36 +0,0 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
-import Router from "Router/Router";
-import { createTheme, ThemeProvider } from "@mui/material/styles";
-
-const theme = createTheme({
-  typography: {
```
Comment (khalt00) [inline src2/App.tsx:8]:
```suggestion
const theme1 = createTheme1({

```

**Pull request closed without merging**